Extension/src/common.ts

```diff
@@ -912,6 +912,26 @@ export async function promptReloadWindow(message: string): Promise<void> {
     }
 }
 
+export async function promptCDDFailed(copy?: boolean): Promise<void> {
+    const copyMessage: string = localize("cdd.copyMessage", "Failed to copy declaration or definition to clipboard.");
+    const createMessage: string = localize("cdd.createMessage", "Failed to create declaration or definition.");
+
+    if (copy) {
```

Comment:
Could you instead use the bool to decide what value to pass to a single call to `showInformationMessage`.  Using the "?:" operators, you could reduce this to from 5 lines to 1 line.